storage/storage.hpp

```diff
@@ -246,6 +247,10 @@ class Storage
   /// \param childrenId is filled with children node ids by a parent. For example GetChildren(GetRootId())
   /// returns in param all countries ids. It's content of map downloader list by default.
   void GetChildren(TCountryId const & parent, TCountriesVec & childrenId) const;
+  /// \param leavesInSubtree is filled with all leaf nodes ids (file names without extension) by a parent.
+  /// If parent is a leaf it is added to leavesInSubtree.
+  /// For example GetChildren(GetRootId()) returns in param all names of mwm files.
```

Comment:
/// For example GetAllLeavesInSubtree(GetRootId()) returns in param all names of mwm files.